mindjunkies/accounts/forms.py

```
@@ -1,7 +1,20 @@
+from django import forms
 from cloudinary.forms import CloudinaryFileField
 from django.forms import ModelForm
+from django.contrib.auth.forms import UserCreationForm
```

Ruff (0.8.2)

`django.contrib.auth.forms.UserCreationForm` imported but unused

Remove unused import: `django.contrib.auth.forms.UserCreationForm`

(F401)